index.js

```diff
@@ -2,19 +2,516 @@ const readline = require('readline');
 const readlineInterface = readline.createInterface(process.stdin, process.stdout);
 
 function ask(questionText) {
-  return new Promise((resolve, reject) => {
-    readlineInterface.question(questionText, resolve);
-  });
+	return new Promise((resolve, reject) => {
+		readlineInterface.question(questionText, resolve);
+	});
 }
 
-start();
+let playerInventory = [];
+
+//assigning variables as undefined in order to make global functions out of the scope of the room's
+
+collectedItem = undefined;
+userAnswer = undefined;
+droppableItems = undefined;
+roomItems = undefined;
+
+/*-----------global take function that will remove item from the room's inventory and place it into the players inventory*/
```

Comment:
Keep this habit of creating these great sectional comments. They grab your attention and get enough basic info across to set reference anchor points for yourself and any team mates.